client/src/app/site/config/components/config-field/config-field.component.ts

```diff
@@ -147,15 +167,14 @@ export class ConfigFieldComponent extends BaseComponent implements OnInit {
     }
 
     /**
-     * Updates the config field.
+     * Sends an update request for the config item to the server.
      * @param value The new value to set.
      */
     private update(value: any): void {
-        // TODO: Fix the Datetimepicker parser and formatter.
-        if (this.configItem.inputType === 'datetimepicker') {
-            value = Date.parse(value);
+        // TODO: Solve this via a custom input form.
+        if (this.configItem.inputType === 'translations') {
+            value = JSON.parse(value);
```

Comment:
Custom translations are already implemented. This will break them.
#4231 https://github.com/OpenSlides/OpenSlides/pull/4231/files#diff-b3ae5e906eddc9aa51e177335ac3e136L160

Reply:
Sorry, this was a botched merge reintroducing old code